coderd/coderdtest/authorize.go

```diff
@@ -358,6 +358,8 @@ func (s *PreparedRecorder) CompileToSQL(ctx context.Context, cfg regosql.Convert
 // Meaning 'FakeAuthorizer' by default will never return "unauthorized".
 type FakeAuthorizer struct {
 	ConditionalReturn func(context.Context, rbac.Subject, policy.Action, rbac.Object) error
+	RegoAuthorizer    *rbac.RegoAuthorizer
+	sqlFilter         string
```

Comment:
Do we include `RegoAuthorizer` only to get the `CompileToSQL` function?

I ask because the `FakeAuthorizer` should really not use the rego authorizer at all.